README.md

```diff
@@ -33,54 +34,54 @@ By default, the prefix is `/lambda/mipsSecret`, resulting the following required
 
 The following template parameters are used to configure CloudFormation resources
 
-| Template Parameter | Description |
-| --- | --- |
-| SsmAliasPrefix | Prepend this value to the Alias Name of the KMS key created. |
-| SsmKeyAdminArn | Grant KMS admin access to these ARNs for managing secure secrets. |
+| Template Parameter | Description                                                       |
+| ------------------ | ----------------------------------------------------------------- |
+| SsmAliasPrefix     | Prepend this value to the Alias Name of the KMS key created.      |
+| SsmKeyAdminArn     | Grant KMS admin access to these ARNs for managing secure secrets. |
 
 The following template parameters are set as environment variables in the lambda environment:
 
-| Template Parameter | Environment Variable | Description |
-| --- | --- | --- |
-| CacheTTL | CacheTTL | TTL value for CloudFront cache objects. |
-| MipsOrganization | MipsOrg | Log in to this organization in the finance system. |
-| SsmParamPrefix | SsmPath | Path prefix for required secure parameters. |
-| CodesToOmit | CodesToOmit | List of numeric codes to treat as inactive. |
-| NoProgramCode | NoProgramCode | Numeric code to use for "No Program" meta-program. |
-| OtherCode | OtherCode | Numeric code to use for "Other" meta-program. |
+| Template Parameter | Environment Variable | Description                                        |
+| ------------------ | -------------------- | -------------------------------------------------- |
+| CacheTTL           | CacheTTL             | TTL value for CloudFront cache objects.            |
+| MipsOrganization   | MipsOrg              | Log in to this organization in the finance system. |
+| SsmParamPrefix     | SsmPath              | Path prefix for required secure parameters.        |
+| CodesToOmit        | CodesToOmit          | List of numeric codes to treat as inactive.        |
+| NoProgramCode      | NoProgramCode        | Numeric code to use for "No Program" meta-program. |
+| OtherCode          | OtherCode            | Numeric code to use for "Other" meta-program.      |
```

Comment:
this table is missing `CodesToAdd`

Reply:
oh, whoops, I forgot to remove that from the template when I removed it from the lambda